git_theta/checkpoints.py

```diff
@@ -137,7 +137,7 @@ def _iterate_dir_leaves(root, prefix):
     return _iterate_dir_leaves(root, [])
 
 
-def get_checkpoint(checkpoint_type: str) -> Checkpoint:
+def get_checkpoint_class_by_name(checkpoint_type: str) -> Checkpoint:
```

Comment:
If we do change this name we need to update the `bin/git-theta` script to use the new name yeah?